### ISSUE TITLE
Support asterisks, *, for starting a todo line

### ISSUE THREAD

**Issue by toastal** [dewyze/vim-tada#1]
Both AsciiDoc and Markdown support starting lists with `(-|\*)`. It'd be ideal-er to support the user's muscle memory.

**Comment by dewyze** [comment]
This morning I tried a quick approach where it would just recognize either character, but it was not that simple. A lot of the logic for adding/removing boxes is more specific, and the pattern of `\(-\|\*\)` (or the magic version or whatever) doesn't always work, because sometimes it needs to function as a capture group, sometimes it juts needs to function as a regex, and sometimes it would need to be the actual character.

I think making it a configurable thing, while not quite as nice, would be the better approach, because then I always know the character to look for.

Feel free to open a PR, or I will try to get to it in a week or 2.

**Comment by toastal** [comment]
That would be fair. Setting a specific sigil could be okay as well. I actually tried to clone to see if I could open a merge request, but got a little lost at some point (I thought it'd be as trivial as modifying the RegExp as well).

**Comment by dewyze** [comment]
@toastal I think I have solution, but I really _hate_ how the testing turned out. Any ideas on a better way to do this besides needing to have `#{SIGIL}` everywhere? To be fair, I haven't though a lot about it. And I don't _really_ want to duplicate all the tests.

I'm wondering if some wrapper around the heredocs for tests that would subout a sigil, and then rake could just run the suite twice with an constant or envvar set.

**Comment by toastal** [comment]
Not really. I was hoping that it was an easy regex which is why I opened the issue :(
